batch/sc-batch-fargate.yaml

```diff
@@ -0,0 +1,200 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: 'AWS Batch with Fargate'
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Container Image
+        Parameters:
+          - Image
+          - Command
+      - Label:
+          default: Synapse
+        Parameters:
+          - SynapsePAT
+      - Label:
+          default: Schedule Job
+        Parameters:
+          - EnableSchedule
+          - Schedule
+Parameters:
+  Image:
+    Type: String
+    Description: >
+      The container image to run, can be from dockerhub or from AWS ECR repository.
+    Default: "debian:latest"
+    ConstraintDescription: >
+      Must be a valid dockerhub or AWS ECR container image
+      Example: debian:latest or 111111111.dkr.ecr.us-east-1.amazonaws.com/MY-IMAGE:latest
+  Command:
```

Comment:
In addition to providing a `Command` parameter, providing a list of environment variables would be useful.  If we don't do this I fear there will be a lot of Docker repo's that can't be used with this product.

Reply:
Env Vars are passed to the container on line 131 but seems like we may need more.

Reply:
We now allow the user to pass an arbitrary list of ENV VARS